src/transformers/models/deberta/configuration_deberta.py

```diff
@@ -82,6 +82,9 @@ class DebertaConfig(PretrainedConfig):
             `["p2c", "c2p"]`.
         layer_norm_eps (`float`, *optional*, defaults to 1e-12):
             The epsilon used by the layer normalization layers.
+        legacy (`bool`, *optional*, defaults to `True`):
+            Whether or not the model should use the legacy `LegacyDebertaOnlyMLMHead`, which does not work properly
+            for mask infilling tasks.
```

Comment:
Does this also support the specific dropout/softmax classes or are we dropping them entirely?

Reply:
We are dropping them completely!